osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -95,7 +97,7 @@ private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes
 
             // Scale accuracy more harshly on nearly-completely mono (single coloured) speed maps.
             double accScalingExponent = 2 + attributes.MonoStaminaFactor;
-            double accScalingShift = 400 - 100 * attributes.MonoStaminaFactor;
+            double accScalingShift = 500 - 100 * attributes.MonoStaminaFactor;
```

Comment:
This was balanced together with https://github.com/ppy/osu/pull/31499 right?

Reply:
Yep, was balanced together.